contracts/v0.8/MarketAPI.sol

```diff
@@ -33,17 +33,10 @@ library MarketAPI {
     using MarketCBOR for *;
 
     /// @notice Deposits the received value into the balance held in escrow.
-    function addBalance(bytes memory provider_or_client) internal {
+    function addBalance(bytes memory provider_or_client, uint256 value) internal {
         bytes memory raw_request = provider_or_client.serializeAddress();
```

Comment:
Might want to validate that there's enough balance to cover the value? Not necessarily since the FVM will fail to execute this send anyway.